server/tserver/src/main/java/org/apache/accumulo/tserver/ScanServer.java

```diff
@@ -280,6 +281,7 @@ public ScanServer(ScanServerOpts opts, String[] args) {
     delegate = newThriftScanClientHandler(new WriteTracker());
 
     this.groupName = Objects.requireNonNull(opts.getGroupName());
```

Comment:
Is this null check still required in addition to validation?

Reply:
ClusterConfigParser.validateGroupNames does not throw an exception if the set is empty, so it's still needed.